tests/conftest.py

```diff
@@ -6,6 +6,7 @@
 import pyarrow as pa
 import pyarrow.parquet as pq
 import pytest
+import zstandard as zstd
```

Comment:
Maybe let's move this import into the zstd_path function ?

This way people that don't have it installed won't have an error when running the tests

Reply:
It is a dependency in "tests": https://github.com/huggingface/datasets/blob/2515d314185ec213d1586e548b45060a29b8dca0/setup.py#L129

Reply:
https://github.com/huggingface/datasets/pull/171/commits/3731c579b6a7a5fb223f2c31615a45a16d601048

Reply:
What if people want to run some tests without having zstandard ?
Usually what we do is add a decorator @require_zstandard for example